sdk/storage/Azure.Storage.Files/src/DirectoryClient.cs

```diff
@@ -307,18 +335,27 @@ public virtual DirectoryClient GetSubdirectoryClient(string subdirectoryName)
                     message: $"{nameof(this.Uri)}: {this.Uri}");
                 try
                 {
-                    return await FileRestClient.Directory.CreateAsync(
+                    smbProperties ??= new FileSmbProperties();
```

Comment:
Why not declare a new value that's not nullable so you don't have to `.Value` everywhere below?

Reply:
Fixed.  But also this is really nit-picky.

Reply:
Yes - this should have been prefixed by `Nit`